From 7842124bb42f072f83c9c292ef95d8d8d0f1c59c Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 18 Mar 2024 15:42:15 +0000
Subject: [PATCH 0973/1085] overlays: i2c-rtc: pcf8563 supports wakeup-source

PCF8563 has alarm functionality, so allow it to accept the
wakeup-source parameter.

See: https://github.com/raspberrypi/linux/issues/6030

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi
+++ b/arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi
@@ -348,6 +348,7 @@
 				<&ds3231>,"wakeup-source?",
 				<&mcp7940x>,"wakeup-source?",
 				<&mcp7941x>,"wakeup-source?",
-				<&m41t62>,"wakeup-source?";
+				<&m41t62>,"wakeup-source?",
+				<&pcf8563>,"wakeup-source?";
 	};
 };
